exporter/skywalkingexporter/go.mod

```diff
@@ -9,7 +9,14 @@ require (
 )
 
 require (
-	cloud.google.com/go v0.81.0 // indirect
+	github.com/open-telemetry/opentelemetry-collector-contrib/internal/coreinternal v0.41.0
```

Comment:
Please, cleanup your go.mod a bit: leave only two sections, with the first being the direct dependencies and the second the indirect ones.